README.md

```diff
@@ -22,8 +23,20 @@ This XBlock does two things.
   * *ps* - probability student makes a mistake even if they does know the skill
   * *threshold* - value which should be reached to make skill be set as learned
 
-- **Course skills list**: course's skills related to every certain problem, default problems list is `["center", "shape", "spread", "x axis", "y axis", "h to d", "d to h", "histogram", "None"]`
-- **Edx Adapt API base URL**: Base URL for Edx Adapt API you're going to work with, e.g. `https://edx-adapt.example.com:443/api/v1`. Ask your Edx Adapt provider for details.
+- **Course skills list**: course's skills related to every certain problem, default problems list is `["center",
+"shape", "spread", "x axis", "y axis", "h to d", "d to h", "histogram", "None"]`
+
+- **Edx Adapt API base URL**: Base URL for Edx Adapt API you're going to work with, e.g.
+`https://edx-adapt.example.com:443/api/v1`. Ask your Edx Adapt provider for details.
+
+- **Edx Adapt Course ID**: Optional parameter which is required only in the case adaptive problems is spread between 
+several course's section. By default Course ID is taken from the Edx e.g. 
+`course-v1:<University>+<Course_name>+<course_run>`. E.g. `course-v1:Stanford+PS01+2017`. In case when 
```

Comment:
Can we drop `course-v1:` completely even for default course id?

To avoid all this confusion and not expose users to unnecessary details.